chord_metadata_service/chord/workflows/wdls/experiments_json.wdl

```diff
@@ -37,7 +37,7 @@ task ingest_task {
             -H "Authorization: Bearer ~{token}" \
             --data "@~{json_document}" \
             "~{katsu_url}/ingest/~{dataset_id}/experiments_json")
-        if [[ "${RESPONSE}" != "204" ]]
+        if [[ "${RESPONSE}" != "201" ]]
```

Comment:
is there a way to get the ingest report no matter what with curl? not sure how WDL works, but it would also be nice to capture it in an `output{}` variable in the WDL file